index.bs

```diff
@@ -740,9 +740,7 @@ focus nodes.
   PREFIX rdfs: &lt;http://www.w3.org/2000/01/rdf-schema#&gt;  
   PREFIX ex: &lt;http://www.example/ns/ex#&gt;
 
-  &lt;&gt; st:hasShapeTreeLocator &lt;#locator&gt; .
-
-  &lt;#locator&gt;
+  &lt;&gt;
```

Comment:
I'm sort of ambivalent about adding a type `st:Locator`.
+: orients reader and clarifies which nodes are what in the locator doc.
-: not needed for processing and optional anythings always lead to FUD somewhere.

An editorial compromise it to state the type in a comment:

> <> # <> is a ShapeTree Locator

Someone who's less conservative than myself about promiscuous tying will probably say "why not add a type arc?" and we can reply "we don't want to imply that it's necessary for processing."

The decision isn't strictly dependent on resolving this PR, but the PR makes it easy to find the spots to make that update if we want to.

Reply:
> -: not needed for processing and optional anythings always lead to FUD somewhere.

I like the idea of keeping it mandatory so that code can rely on it's existence and use that as a reliable and simple check for whether or not the document is declaring itself as a shape tree locator, without having to go straight to schema validation. It does also make it a bit more human readable when you have eyeballs directly on the resource.

Reply:
i can live with that by explaining to myself that it's a form of future-proofing.